zipline/utils/tradingcalendar.py

```diff
@@ -14,14 +14,22 @@
 # limitations under the License.
 import pandas as pd
 import pytz
+import warnings
 
 from datetime import datetime
 from dateutil import rrule
 from functools import partial
 
+from zipline.zipline_warnings import ZiplineDeprecationWarning
+
 # IMPORTANT: This module is deprecated and is only here for temporary backwards
 # compatibility.  Look at the `zipline.utils.calendars.trading_schedule`
 # module, as well as the calendar definitions in `zipline.utils.calendars`.
+warnings.warn(
+    "The `tradingcalendar` module is deprecated.",
```

Comment:
deprecation warnings should always tell the user what the new feature is that deprecates the old feature. Can we add something like, `see 'zipline.utils.calendar' for the trading calendar API`